src/utils/styledLink.js

```diff
@@ -0,0 +1,15 @@
+
+import styled from "styled-components";
+
+const StyledLink = styled.button`
+    font-family: "Sofia Pro Soft Regular";
+    font-size: 13;
+    text-transform: uppercase;
+    text-decoration: underline;
+    letter-spacing: .05rem;
+    border: none;
+    background: transparent;
+    color: MediumBlue;
+`;
+
+export { StyledLink };
```

Comment:
In the future, we should fix this to just be CSS if possible. Importing styled components package just for changing style of link is unnecessary.

Reply:
Ok - got it. I was thinking though that more components could be styled using the styled-components package, so we could get more usage out of it. We can talk about that change together?